tests/IntegrationTestCase.php

```diff
@@ -34,4 +34,9 @@ public function createWorkspaceYml(string $contents): void
     {
         $this->workspace()->put('workspace.yml', $contents);
     }
+
+    public function removeAnsiColorEscapes(string $content)
+    {
+        return preg_replace('/\x1b\[[0-9;]*m/', '', $content);
```

Comment:
was pretty sure there was an SF "helper" or static method to strip this, can't find it tho

Reply:
an alternative would be to change `workspaceCommand` to pass `--no-ansi`

Reply:
`my127/Console` doesn't have this switch, ANSI color escapes are hardcoded into the `ContextualHelperPlugin`. I might raise a separate PR for that in the future though, since it always bothered my bash completion function I use.